pkg/controller/integrationplatform/initialize.go

```diff
@@ -70,7 +70,11 @@ func (action *initializeAction) Handle(ctx context.Context, platform *v1.Integra
 	}
 
 	if platform.Status.Build.PublishStrategy == v1.IntegrationPlatformBuildPublishStrategyKaniko {
```

Comment:
You can comment this block with `// nolint: staticcheck` to avoid lint complains (due to the deprecation)